### PR TITLE
Missing protocol in the mocked kubernetes environment for the Stork Kubernetes quickstart

### PR DESCRIPTION
**Pull request by cescoffier**
\CC @gsmet

### CODE REVIEW

File: kafka-quickstart/processor/pom.xml [hunk 1]
```diff
@@ -34,6 +34,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-reactive-messaging-kafka</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-health</artifactId>
+        </dependency>
```
Comment (gsmet) [inline kafka-quickstart/processor/pom.xml:40]: I think this change is unrelated? I don't mind adding it if it's not a mistake.

Comment (aureamunoz) [reply kafka-quickstart/processor/pom.xml:40]: Yes, I have the same question

Comment (cescoffier) [reply kafka-quickstart/processor/pom.xml:40]: hum.... sorry, I didn't stage that but the github cli did. Let me remove it.
